src/v3/test/integration/authenticator-verification-email.test.tsx

```diff
@@ -20,11 +19,15 @@ import sessionExpiredResponse from '../../src/mocks/response/idp/idx/identify/er
 
 describe('Email authenticator verification when email magic link = undefined', () => {
   describe('renders correct form', () => {
-    beforeEach(() => {
+    // Mock system time for triggering resend email reminder element
+    let mockSystemTimeStamp = Date.now();
+    Date.now = jest.fn(() => mockSystemTimeStamp);
```

Comment:
Using jest.useFakeTimers('modern') here breaks our integration test and causes tests to fail.  There seems to be an issue/conflict with the 'modern' fake timer and async calls.  Upgrading to newer jest version did not resolve the issue so it seems like a deeper issue that requires more investigation.  Possibly related info:
https://github.com/facebook/jest/issues/11103
https://github.com/facebook/jest/issues/13018

So I opted to mock the Date.now method here for now since we have more control and behavior is more predictable.